azure-communication-ui/calling/src/main/java/com/azure/android/communication/ui/calling/presentation/CallCompositeActivity.kt

```diff
@@ -253,10 +253,9 @@ internal class CallCompositeActivity : AppCompatActivity() {
                 supportActionBar?.hide()
                 requestedOrientation =
                     when {
+                        (configuration.callScreenOrientation != null) -> getScreenOrientation(configuration.callScreenOrientation)
```

Comment:
check if getScreenOrientation() returns not nil, do not rely on configuration.callScreenOrientation != null, as configuration.callScreenOrientation may have a value and still be not possible to convert